.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+nps care
```

Comment:
Added this, you should be able to get it by running:
1. `yarn`
2. `npx husky install`